pypesto/visualize/waterfall.py

```diff
@@ -23,6 +93,7 @@ def waterfall(
     scale_y: Optional[str] = 'log10',
     offset_y: Optional[float] = None,
     start_indices: Optional[Union[Sequence[int], int]] = None,
+    n_starts_to_zoom: int = 10,
```

Comment:
```suggestion
    n_starts_to_zoom: int = 0,
```

Perhaps the default should be not to zoom in?